### ISSUE TITLE
Keras tutorial is waiting on an internal Keras modification to execute properly

### ISSUE THREAD

**Issue by npapernot** [tensorflow/privacy#21]
We are waiting for Keras' `compile` method to accept a non-scalar loss so the corresponding tutorial can run without making changes to the tensorflow library.

See b/124011218

**Comment by npapernot** [comment]
Closed by #3c1e9994ebe29fffb8c97f952fefa5b917d92529
